gnomad/utils/file_utils.py

```diff
@@ -222,3 +222,52 @@ def repartition_for_join(
             " partitions than the original HT!"
         )
     return ht._calculate_new_partitions(ht.n_partitions() * new_partition_percent)
+
+
+def create_vds(
```

Comment:
I'd like to leave it as create_vds since well expand the function to accepting the VDS param in the future and if we change the name to gvcf, well have a breaking change or need to create a new function all together